client/src/pages/CreateSurvey/index.jsx

```diff
@@ -43,10 +43,13 @@ const CreateSurvey = ({ history }) => {
     modalStyle,
     isConfirming,
   } = useSelector((state) => state.createSurveyReducer);
+
+  const { recipientIds } = useSelector((state) => state.employeeTableReducer);
   const dispatch = useDispatch();
 
   useEffect(() => {
     dispatch({ type: 'RESET_CS_MODAL_STATE' });
```

Comment:
CS (in the type) isn't clear to me what it means.

Reply:
This is a very useful review comment @TKostrzewski ! I don't know what this means either. Verbose variable / constant names are perfectly acceptable